subcommands/targets/show.go

```diff
@@ -4,12 +4,14 @@ import (
 	"encoding/base64"
 	"fmt"
 	"os"
+	"strconv"
 	"strings"
 
 	"github.com/cheynewallace/tabby"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"github.com/theupdateframework/notary/tuf/data"
```

Comment:
Can you give this module some sensible alias e.g. `tuf` (like we do elsewhere)?
It's hard to understand in the code what `data` stands for.

Reply:
https://github.com/foundriesio/fioctl/pull/109/commits/553ab17170e31726a938ad5c920191deb4d1f4ac